CHANGELOG.rst

```diff
@@ -4,6 +4,24 @@ Release Notes
 Changes are ordered reverse-chronologically.
 
 
+0.10.3
+------
+
+Internationalization and localization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Added Mexican Spanish (es_MX) and Bulgarian (bg)
+
+Fixed
+~~~~~
+
+ - Upgrade issue upon username conflict between device owner and facility user
+ - Channel import listing of USB devices when non-US locale
+ - Content counts for coaches would in some cases be wrongly displayed
```

Comment:
Maybe `Counts for coach-specific content`

Reply:
Thanks, fixed!